pkgs/servers/code-server/default.nix

```diff
@@ -12,29 +13,35 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "code-server";
-  version = "3.6.0";
-  commit = "a4a03c14922ccaec2a9ff8d1b7b2af8522a4214d";
+  version = "3.8.0";
+  commit = "c4610f7829701aadb045d450013b84491c30580d";
 
   src = fetchFromGitHub {
     owner = "cdr";
     repo = "code-server";
     rev = "v${version}";
-    sha256 = "1c0p1s0bl3az5ysl97mz3gbynyndz6jd2jj7lx2snz6jqqd43y9p";
-    fetchSubmodules = true;
+    sha256 = "1snc7dbqfz53337h6av2zhkrn54ypanxljs5by4jqczq96c2v6yk";
   };
 
   cloudAgent = buildGoModule rec {
     pname = "cloud-agent";
-    version = "0.1.0";
+    version = "0.2.1";
+    commit = "5beac91dd5feab9c745d58fda8118a0efaa1ef74";
```

Comment:
Why do we need this? I seems to be unused.

Reply:
👍 Deleted this